autodocodec-api-usage/test/Autodocodec/Orphans.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Autodocodec.Orphans where
+
+import Data.Aeson (FromJSON, ToJSON)
+import Data.Functor.Const (Const (Const))
+import Data.Functor.Identity (Identity (Identity))
+import Data.Monoid (Alt, Dual)
+import qualified Data.Monoid as Monoid
+import qualified Data.Semigroup as Semigroup
+import Test.Syd.Validity
+
+deriving newtype instance (Validity a) => Validity (Identity a)
```

Comment:
These need to be upstreamed to `validity` and `genvalidity` at https://github.com/NorfairKing/validity first.

Reply:
@NorfairKing I've created a PR for this here: https://github.com/NorfairKing/validity/pull/119

Reply:
Merged and released.